packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const accountHistoryCount = await client.account.accountHistoryCount()
+
+        expect(typeof accountHistoryCount).toBe('number')
+        expect(accountHistoryCount).toBeGreaterThan(0)
+      })
+    })
```

Comment:
alignment

Reply:
@canonbrother , the alignment on line 527 and 528? 

<img width="755" alt="image" src="https://user-images.githubusercontent.com/57817584/119003557-b8f84780-b985-11eb-9710-6c242feaefaf.png">

Reply:
gosh.. lol.. sorry... you're right... i gonna rest